fix(svgs): convert to react components and update usage docs (#1733)
carbon-design-system/carbon-addons-iot-react: .github/CONTRIBUTING.MD

the sass partial approach. This helps promote consistency with the Carbon Design
System.

## Adding new SVG images

If SVG files need to be added that are not already distributed by Carbon, please
take the following steps.

1. first add the SVG files to `src/icons/svg`.

> 2. Next, run `npx @svgr/cli src/icons/svg/your-svg-filename.svg`. This will
>    produce a React JSX component file which will then need to be moved to
>    `src/icons/components`.

Reviewer: ```suggestion
2. Next, run `yarn svg-convert`. This will produce a React JSX component 
   file in `src/icons/components`.
```

Author: Updated text
